src/components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState, FormEvent, ChangeEvent, useMemo } from 'react';
+import { Link } from 'react-router-dom';
+
+import { FormInput } from '../FormInput/FormInput';
+import style from './RegistrationForm.module.scss';
+
+type FormData = {
```

Comment:
`type FormData = typeof INITIAL_FORM_DATA` could work as well. Depends what do you want to have as the definition